draft-ietf-webpush-protocol.xml

```diff
@@ -305,7 +305,22 @@
         </t>
       </section>
     </section>
-
+    <section title="Connecting to the Push Service">
+      <t>
+        The push service shares the same default port number (443/TCP) with HTTPS, but MAY
+        also advertise the IANA allocated TCP port 1001 using HTTP alternative services
+        <xref target="I-D.ietf-httpbis-alt-svc"/>:
+      </t>
+      <t>
+        The configuration of this port can potentially be optimized for better timeout
+        characteristics for push notifications than port 443.
+      </t>
+      <t>
+        NAT(s) MUST comply with REQ-5 in <xref target="RFC7231"/> which requires that
+        "the value of the 'established connection idle-timeout' MUST NOT be less than
+        2 hours 4 minutes".

```

Comment:
This seems a little too optimistic to me.  I don't think that we want to attempt to levy any requirements on middleboxes.  Certainly, I would avoid any MUST-strength language.

I change the emphasis here to the facts.

1. Because port 443 is most often used for web surfing, it can be given a lower timeout in middleboxes than other ports.
2. We want middleboxes to give push an increased timeout duration.  

Then add appropriate encouragement "Middleboxes are encouraged to (or SHOULD, but it's not an interoperability requirement) increase timeouts for connections to port <TBD> on the expectation that data exchange on connections will be infrequent."

The point of this section to strongly "suggest" that offering a push service over port 443 has advantageous reachability characteristics, while noting that it could have terrible keep-alive costs.  Advertising an alternative service on <the IANA-allocated port> allows those clients that are able to take advantage of the timeout characteristics that networks might offer for push.